.github/workflows/configs_sync.yaml

```diff
@@ -0,0 +1,23 @@
+name: Sync configuration files
+
+on:
+  schedule:
+    - cron:  "0 0 * * *"
+  push:
+    branches:
+      - 'master'
+      - 'ci/configs-sync'
```

Comment:
This can be removed. I had it only for testing it in bee-js and forgot to remove it before merge 🙈

```suggestion
```